ppml/services/easy-kms/docker/Dockerfile

```diff
@@ -0,0 +1,49 @@
+ARG BIGDL_VERSION=2.3.0-SNAPSHOT
```

Comment:
organize the dockerfiles into base and ref folders
one readme for bigdl-kms
two bigdl-kms-depolyment for both SGX and TDX
rename start-service to bigdl-kms-entrypoint.sh